.npmignore

```diff
@@ -0,0 +1,14 @@
+bower_components/
+tests/
+tmp/
+dist/
```

Comment:
adding `dist`/ to npmignore could cause problems for anybody who's `npm install`ing but then using the dist directory... would prefer to leave this in!